package.json

```diff
@@ -2,6 +2,7 @@
   "name": "pdf.js",
   "version": "0.8.0",
   "devDependencies": {
+    "gulp": "^3.9.0",
```

Comment:
Let's change this to `"^3.9.1"` to indicate that we can use the latest version (just like we do below this line).